src/app/newmarket/page.tsx

```diff
@@ -79,27 +81,88 @@ const NewmarketForm: React.FC = () => {
     return null;
   }
 
+   const styleBox = css`
+  margin: 0 auto;
+  padding: 50px;
+  background-color: white;
+  border-radius: 8px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+`;
```

Comment:
please fix the indentation(including this one and every one follows)

Reply:
done